lib/test_nplusone.py

```diff
@@ -12,6 +12,7 @@
 
 nfiles = int(config.get('nplusone_nfiles',10))
 filesize = config.get('nplusone_filesize',1000)
+measurement = config.get('monitoring_push',"cernbox.cboxsls")
```

Comment:
is push mandatory for this test to run ? I'd prefer if there is a `None` value and we don't push anything in that case instead of risking pushing to a remote server.